content/docs/alerting/configuration.md

```diff
@@ -303,8 +303,10 @@ PagerDuty provides documentation on how to integrate [here](https://www.pagerdut
 # Whether or not to notify about resolved alerts.
 [ send_resolved: <boolean> | default = true ]
 
-# The PagerDuty service key.
-service_key: <tmpl_secret>
+# The PagerDuty integration key.
+# When using the `routing_key` form, the PagerDuty integration must be of type `Events API v2`
+# To use the PagerDuty integration type `Prometheus`, use `service_key: <tmpl_secret>` instead
```

Comment:
All fields should be listed directly, not in comments.

@stuartnelson3

Reply:
Better?